### PR TITLE
Prepare contributing guide

### PR DESCRIPTION
**Pull request by rnbguy**
ref #101

### CODE REVIEW

**Comment by rnbguy** [comment]
I added a prelim version of the guide. It may not be verbose :sweat_smile: 

Let me know if I missed any topic or if you do not agree with something.

**Comment by rnbguy** [comment]
@andrey-kuprianov do you think we should init `unclog` with a summary of previous change log?

**Comment by rnbguy** [comment]
[Github Doc](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors) mentions

> To help your project contributors do good work, you can add a file with contribution guidelines to your project repository's root, `docs`, or `.github` folder.

Moving the file to `.github` directory.

**Comment by rnbguy** [comment]
Also, a PR template would be great addition.

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+We follow [_Semantic Versioning_](http://semver.org) for releases.
+
+When all the code changes are ready and merged to `main`, a `release/vX.Y.Z` branch must be opened from `main`.
+All the changes for bumping version numbers and moving unreleased changes via unclog should be committed in this branch.
+
+When `release/vX.Y.Z` is ready with correct version bumps, create a PR and merge it to `main`.
+Tag the merge commit with `vX.Y.Z` and prepare a release on Github, publish on package registries for Golang, Rust and Python.
+
+In future, the releases would be done via GH workflow. It must be triggered by tagging a commit with `vX.Y.Z`.
```
Comment (danwt) [inline .github/CONTRIBUTING.md:51]: Is this git tagging like [this](https://git-scm.com/book/en/v2/Git-Basics-Tagging) or does it mean simply having vX.Y.Z somewhere in the commit title/description?

Comment (danwt) [reply .github/CONTRIBUTING.md]: Awesome

**Pull request merged**